index.html

```diff
@@ -27,13 +27,119 @@
             </button>
         </div>
         <div class = 'headline'>
-            <p>Eduardo Hernández</p>
-            <p>Frontend Developer</p>
+            <h1>Eduardo Hernández</h1>
+            <h3>Frontend Developer</h3>
             <p>Frontend developer looking for ideas and projects for himself and others</p>
         </div>
+        <div class = 'boxcode'>
+            <p>let button_menu = document.querySelector(".header_button");<br>
+            let float_menu  = document.querySelector(".header_menu--float");<br>
+            <br>
+            let line_top   = document.querySelector(".header_button_line--<br>
+            top");<br>
+            let line_under = document.querySelector(".header_button_line--<br>
+            bottom");<br>
+            <br>
+            button_menu.isSelected = false;<br>
+            <br>
+            function addEffectWriting(element, time) {<br>
+            &nbsp;let element_array = element.innerText.split("");<br>
+            <br>
+            &nbsp;element.count     = 0;<br>
+            &nbsp;element.innerText = "";<br>
+            <br>
+            &nbsp;function writeSymbols() {<br>
+            &nbsp;&nbsp;element.innerText += element_array[element.count];<br>
+            &nbsp;&nbsp;element.count++;<br>
+            <br>
+            &nbsp;&nbsp;if (element.count &lt;= element_array.length) {<br>
+            &nbsp;&nbsp;&nbsp;setTimeout(() => {<br>
+            &nbsp;&nbsp;&nbsp;&nbsp;writeSymbols();<br>
+            &nbsp;&nbsp;&nbsp;}, time);<br>
+            &nbsp;&nbsp;} else if (element.count >= element_array.length) {<br>
+            &nbsp;&nbsp;&nbsp;element.count     = 0;<br>
+            &nbsp;&nbsp;&nbsp;element.innerText = "";<br>
+            <br>
+            &nbsp;&nbsp;&nbsp;writeSymbols();<br>
+            &nbsp;&nbsp;}<br>
+            &nbsp;}<br>
+            &nbsp;writeSymbols();<br>
+            } ...</p>
+        </div>  
+        <div class="photo">
+          <div class="photolayer1"></div>
+          <div class="photolayer2"></div>
+          <div class="photolayer3"></div>
+        </div>
+      </div>
     </header>
     <main>
-
+      <div class="aboutme">
+        <div class="polygonicon">
+          <img src="./images/pilygonicon.png" alt="icon">
+        </div>
+        <p>A few years ago, I became interested in developing websites and interfaces.
+        Since then, I have been steadily improving my skills. Now I am developing websites
+        and web applications to order or for myself.</p>
+      </div>
+      <div class="myskills">
+        <h2>My Skills</h2>
+        <div class="skill">
+          <img src="./images/icon html.png" alt="HTML5 icon">
+          <img class="top" src="./images/icon html -- hover.png" alt="HTML5 icon hover">
```

Comment:
I was also seeing that you have a couple of imgs that contain the same image, I suggest you use an svg image so you can change the color of this icon

Reply:
Hi @Albinpirela thanks for the suggestion, i'll change that. 👍